lib/Backend/GlobOpt.h

```diff
@@ -1007,6 +1011,11 @@ class GlobOptBlockData
     SListBase<IR::Opnd *> *                 callSequence;
     StackLiteralInitFldDataMap *            stackLiteralInitFldDataMap;
 
+    CapturedValues *                        capturedValuesCandicate;
```

Comment:
Right, will fix this one.

---
In reply to: [67772925](https://github.com/Microsoft/ChakraCore/pull/1152#discussion_r67772925) [](ancestors = 67772925)